db/re/status.yml

```
@@ -9045,3 +9045,93 @@ Body:
       NoClearance: true
       NoDispell: true
       NoRemoveOnDead: true
+  - Status: Contents_1
+    Icon: EFST_CONTENTS_1
+    CalcFlags:
+      All: true
```

Status only affects SCB_BASE
```suggestion
      Base: true
```